app/models/user.rb

```diff
@@ -219,7 +219,7 @@ def assignable_roles
 
   def logs_filters(specific_org: false)
```

Comment:
I'd probably have a separate `bulk_uploads_filters` method, cause the naming of this one is a bit misleading. Also it feels like the list of filters for bu wouldn't necessarily depend on the user type so I'm not sure if it's the best place for this to be. 
I think this is wrong for schemes as well, so wasn't introduced by you. 
It's up to you if you want to update it